wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -555,6 +556,7 @@ export default {
             .catch((e) => {
               console.error('Web3 contract.transfer method - error', e);
               this.error = `${this.$t('exoplatform.wallet.error.emptySendingTransaction')}: ${truncateError(e)}`;
+              this.showAlert('error',this.$t('exoplatform.wallet.error.emptySendingTransaction'));
```

Comment:
do you mean that we will display the error twice in `this.error` and inside a snackbar ? BTW, the label key isn't ok, it doesn't seems about a transaction failure/error.